src/main/java/com/yelp/nrtsearch/server/luceneserver/search/query/multifunction/MultiFunctionScoreQuery.java

```diff
@@ -53,6 +55,8 @@ public class MultiFunctionScoreQuery extends Query {
   private final FilterFunction[] functions;
   private final FunctionScoreMode scoreMode;
   private final BoostMode boostMode;
+  private final float minScore;
```

Comment:
Using the min wrapper requires handling in other methods. See https://github.com/elastic/elasticsearch/blob/v7.2.0/server/src/main/java/org/elasticsearch/common/lucene/search/function/FunctionScoreQuery.java for examples. I see references in `createWeight`, `explain`, 'isCacheable`, `equals`, and `hashCode`.

Reply:
added accordingly.
I manually changed `toString`. But `hashCode` `equals` are generated by Intellij, so they might be slightly different.